### PR TITLE
[567249] Unable to use filtering in Trace Manager during selection of trace element

### PR DESCRIPTION
**Pull request by claudiasima-ro**
Added icons and changed editor for traces

### CODE REVIEW

File: core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/plugin.properties [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-# Contributors:
-#    Thales - initial API and implementation
-# =============================================================================
-#Properties file for org.polarsys.capella.core.platform.eclipse.ui.trace
-providerName = Eclipse.org
-pluginName = MDTrace Eclipse UI For Capella
\ No newline at end of file
+# =============================================================================
+# Copyright (c) 2006, 2015 THALES GLOBAL SERVICES.
+#
```
Comment (georgiana-ecobici) [inline core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/plugin.properties:3]: update header, change the last modification from 2015 to 2020.

Copyright (c) 2006, 2020 THALES GLOBAL SERVICES. ....

File: core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/src/org/polarsys/capella/core/platform/eclipse/capella/ui/trace/messages/messages.properties [hunk 2]
```diff
@@ -1,61 +1,58 @@
...
-TracePageContentProvider.page.title=Capella trace
-TracePageContentProvider.page.description=This editor displays Capella trace.
-
-# The tab strings.
-TraceGeneralTabDescriptor.tab.title=&General
-TraceViewer.label.summary=Summary
-TraceViewer.label.description=Description
-
+#===============================================================================
+# Copyright (c) 2006, 2018 THALES GLOBAL SERVICES.
```
Comment (georgiana-ecobici) [inline core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/src/org/polarsys/capella/core/platform/eclipse/capella/ui/trace/messages/messages.properties:2]: unexpected changed lines, change only the line affected.

File: core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/src/org/polarsys/capella/core/platform/eclipse/capella/ui/trace/views/components/TraceTreeViewer.java [hunk 3]
```diff
@@ -294,6 +300,8 @@ public Composite getControl(Composite parent_p) {
     layoutData.verticalAlignment = GridData.FILL;
     _treeViewer.getControl().setLayoutData(layoutData);
 
+    
+
```
Comment (georgiana-ecobici) [inline core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/src/org/polarsys/capella/core/platform/eclipse/capella/ui/trace/views/components/TraceTreeViewer.java:304]: remove extra space.

File: core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/src/org/polarsys/capella/core/platform/eclipse/capella/ui/trace/views/components/TraceTreeViewer.java [hunk 4]
```diff
@@ -482,4 +491,61 @@ public void updateComponent(TraceableElement newNamedElement_p) {
     _treeViewer.expandAll();
     setComboInput();
   }
+  
+  
+  
+///////////////////////DELETE THIS!  
+
```
Comment (georgiana-ecobici) [inline core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/src/org/polarsys/capella/core/platform/eclipse/capella/ui/trace/views/components/TraceTreeViewer.java:498]: delete this :)

File: core/plugins/org.polarsys.capella.core.platform.sirius.ui.actions/META-INF/MANIFEST.MF [hunk 5]
```diff
@@ -1,30 +1,29 @@
...
- org.polarsys.capella.core.platform.sirius.ui.menu
-Export-Package: org.polarsys.capella.core.platform.sirius.ui.actions,
- org.polarsys.capella.core.platform.sirius.ui.commands,
- org.polarsys.capella.core.platform.sirius.ui.handlers,
- org.polarsys.capella.core.platform.sirius.ui.preferences,
- org.polarsys.capella.core.platform.sirius.ui.session
-Bundle-ActivationPolicy: lazy
-Bundle-Activator: org.polarsys.capella.core.platform.sirius.ui.actions.CapellaActionsActivator
-Bundle-Localization: plugin
+Manifest-Version: 1.0
```
Comment (georgiana-ecobici) [inline core/plugins/org.polarsys.capella.core.platform.sirius.ui.actions/META-INF/MANIFEST.MF:1]: only changed lines shall appear as changed.

File: core/plugins/org.polarsys.capella.core.platform.sirius.ui.actions/plugin.xml [hunk 6]
```diff
@@ -1,716 +1,709 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<?eclipse version="3.2"?>
-<!--
```
Comment (georgiana-ecobici) [inline core/plugins/org.polarsys.capella.core.platform.sirius.ui.actions/plugin.xml:3]: change only affected lines

File: core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/plugin.xml [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    </extension>
+   
+    <extension point="org.eclipse.ui.menus">
+      <menuContribution
+        locationURI="popup:org.polarsys.capella.core.wizards?after=additions">
+        <command  commandId="org.polarsys.capella.core.platform.sirius.ui.actions.wizard.traceManagerCommand" >  
+          <visibleWhen
+            checkEnabled="false">
+              <reference
+                definitionId="org.polarsys.capella.core.platform.sirius.ui.actions.TraceManager.expression">
```
Comment (minhtutonthat) [inline core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/plugin.xml:34]: Could you also move the expression to the plugin org.polarsys.capella.core.platform.eclipse.ui.trace?

File: core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/src/org/polarsys/capella/core/platform/eclipse/capella/ui/trace/views/components/TraceTreeViewer.java [hunk 8]
```diff
@@ -12,9 +12,14 @@
  *******************************************************************************/
 package org.polarsys.capella.core.platform.eclipse.capella.ui.trace.views.components;
 
+import java.awt.Dimension;
```
Comment (minhtutonthat) [inline core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/src/org/polarsys/capella/core/platform/eclipse/capella/ui/trace/views/components/TraceTreeViewer.java:15]: Are all of these imports necessary?

**Review by minhtutonthat — CHANGES_REQUESTED** [review]

File: core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/META-INF/MANIFEST.MF [hunk 9]
```diff
@@ -20,3 +20,5 @@ Export-Package: org.polarsys.capella.core.platform.eclipse.capella.ui.trace,
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-Activator: org.polarsys.capella.core.platform.eclipse.capella.ui.trace.MDTrace
 Bundle-Localization: plugin
+Import-Package: org.polarsys.capella.core.ui.properties.wizards
```
[the remark this answers is not in the document]
Comment (claudiasima-ro) [reply core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/META-INF/MANIFEST.MF]: yes

File: core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/src/org/polarsys/capella/core/platform/eclipse/capella/ui/trace/views/components/TraceTreeViewer.java (hunk 8 above)
Comment (claudiasima-ro) [reply core/plugins/org.polarsys.capella.core.platform.eclipse.ui.trace/src/org/polarsys/capella/core/platform/eclipse/capella/ui/trace/views/components/TraceTreeViewer.java]: No, I removed them. Thanks!

**Review by minhtutonthat — APPROVED** [review]

**Pull request merged**